examples/todo.rb

```diff
@@ -31,7 +31,7 @@ def complete
   attr_reader :id, :name
 end
 
-class ToDoList < Eventory::AggregateRoot
+class ToDoList < Eventory::Domain::AggregateRoot
```

Comment:
I'd prefer this at the top level, along with the repository.

Reply:
Yeah, makes sense. Done in 911f224.